pkg/tsdb/mysql/mysql.go

```diff
@@ -20,10 +21,16 @@ func init() {
 func newMysqlQueryEndpoint(datasource *models.DataSource) (tsdb.TsdbQueryEndpoint, error) {
 	logger := log.New("tsdb.mysql")
 
+	var protocol string
```

Comment:
Please simplify this by initializing protocol to `tcp` and skip the else clause:

```go
protocol := "tcp"
if strings.HasPrefix(datasource.Url, "/") {
	protocol = "unix"
}
```